spec/features/admin/dashboard_spec.rb

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
+
 require "rails_helper"
 require "pp"
 
-feature "Admin - Dashboard" do
```

Comment:
Now everything is `describe`?

Reply:
I guess so 😅 I believe that was a side-effect of when I tried rubocop-rspec and tried to autocorrect as much as possible. I then moved away from rubocop-rspec because there were just too many issues with it (more than 2000 offenses 😢). So, maybe in the future we add it.